packages/ui/src/lib/layouts/BasePage.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+import { createEventDispatcher } from 'svelte';
+
+import { LinearProgress } from '..';
+import CloseButton from '../button/CloseButton.svelte';
+import Link from '../link/Link.svelte';
+
+export let title: string;
+export let titleDetail: string | undefined = undefined;
+export let subtitle: string | undefined = undefined;
+export let breadcrumbLeftPart: string | undefined = undefined;
+export let breadcrumbRightPart: string | undefined = undefined;
+export let breadcrumbTitle: string | undefined = '';
+export let hasClose: boolean = true;
+export let inProgress: boolean = false;
+
+let showBreadcrumb = breadcrumbLeftPart ?? breadcrumbRightPart;
+
+const dispatchClose = createEventDispatcher<{ close: undefined }>();
```

Comment:
We are not suppose to use dispatcher with svelte 5

Reply:
This PR is merging the dispatcher from two components into a single parent, IMHO that would make reviewing harder and should be a separate PR.